apps/pie-microsite/src/_data/normalizeTokens.js

```diff
@@ -0,0 +1,33 @@
+const pieDesignTokens = require('@justeat/pie-design-tokens/dist/tokens.json');
+
+// clone tokens to avoid mutating the original reference.
```

Comment:
Is this necessary if the original import comes directly from a file and is only used once?

Reply:
are you referring to the cloning bit? if so, I think yes, even if the file is used once (so far) and imported from a file. the object will still have the same reference in memory of the original import and any mutation that happens will be carried to the original tokens.json. 

here is a [sandbox](https://codesandbox.io/s/laughing-hamilton-4dtmkt?file=/src/index.js) with some of the issues we might face down the road where the normalization layer leaks its mutation to other modules if it wasn't cloned.

Reply:
Well that's not ideal!